pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -700,6 +701,12 @@ class SubDatetime(datetime):
         expected = Timestamp(2000, 1, 1)
         assert result == expected
 
+    @pytest.mark.skipif(not compat.PY38, reason="Requires python 3.8 and above")
```

Comment:
The "reason" here doesn't tell the reader anything they can't figure out without it, so try to be more specific. e.g. "fromisocalendar was added in stdlib datetime in py38".